src/widget/loginscreen.cpp

```diff
@@ -31,6 +31,16 @@
 #include <QMessageBox>
 #include <QDebug>
 
+#ifdef Q_OS_WIN32
+    #include <windows.h>
+#else
+    #include <X11/XKBlib.h>
```

Comment:
@sudden6 also, it would need a decision if there was a disagreement whether dep should be introduced.

There's little to no point in discussing/voting on something that ~everyone agrees on.
With that being said, I have ~no opinion on this one – provided that code works on supported platforms, there are docs, etc. Generally, what I want is "get it to work".

If there is something that you perceive to be an issue, just simply state it – if it's something that simply needs to be fixed, then it needs to be fixed. If it's something that depends on ~opinion, there might be someone disagreeing, in which case that would need to be discussed/voted on, or no one would, in which case just proceed with whatever needs to be done.

And if there is something that you want opinions of someone else before proceeding – IMO there's little to no point in waiting for an official meeting to get it – just simply get it whenever you have a chance. Now, if the only chance of getting opinion of e.g. @tux3 on some matter is the meeting, that's a separate problem…